syntaxes/starlark.tmLanguage.json

```diff
@@ -0,0 +1,929 @@
+                }
+            ]
+        },
+        "function-call-name": {
+            "patterns": [
+                {
+                    "include": "#type-identifier"
+                },
+                {
+                    "include": "#builtin-function"
+                },
+                {
+                    "name": "entity.name.function.starlark",
+                    "match": "\\b([[:alpha:]_]\\w*)\\b"
+                }
+            ]
+        },
+        "function-arguments": {
+            "begin": "(?:(\\()(?:\\s*(\\*\\*|\\*))?)",
+            "end": "(?=\\))(?!\\)\\s*\\()",
+            "beginCaptures": {
+                "1": {
+                    "name": "punctuation.definition.arguments.begin.starlark"
+                },
+                "2": {
+                    "name": "keyword.operator.unpacking.arguments.starlark"
+                }
+            },
+            "contentName": "meta.function-call.arguments.starlark",
+            "patterns": [
+                {
+                    "match": "(?:(,)(?:\\s*(\\*\\*|\\*))?)",
+                    "captures": {
+                        "1": {
+                            "name": "punctuation.separator.arguments.starlark"
+                        },
+                        "2": {
+                            "name": "keyword.operator.unpacking.arguments.starlark"
+                        }
+                    }
+                },
+                {
+                    "include": "#illegal-name"
+                },
+                {
+                    "match": "\\b([[:alpha:]_]\\w*)\\s*(=)(?!=)",
+                    "captures": {
+                        "1": {
+                            "name": "meta.parameter.keyword.starlark"
+                        },
+                        "2": {
+                            "name": "keyword.operator.assignment.starlark"
+                        }
+                    }
+                },
+                {
+                    "name": "keyword.operator.assignment.starlark",
+                    "match": "=(?!=)"
+                },
+                {
+                    "include": "#expression"
+                },
+                {
+                    "match": "\\s*(\\))\\s*(\\()",
+                    "captures": {
+                        "1": {
+                            "name": "punctuation.definition.arguments.end.starlark"
+                        },
+                        "2": {
+                            "name": "punctuation.definition.arguments.begin.starlark"
+                        }
+                    }
+                }
+            ]
+        },
+        "builtin-function": {
+            "patterns": [
+                {
+                    "name": "support.function.starlark",
+                    "match": "(?<!\\.)\\b(all|any|bool|dict|dir|enumerate|getattr|hasattr|hash|int|len|list|load|max|min|print|range|repr|reversed|sorted|str|tuple|type|zip)\\b"
```

Comment:
`load` can be removed, since it matches `builtin-constant`?

Reply:
`load` is here so that those statements have their arguments colored consistently with other functions (which makes a difference when you use aliasing via keyword arguments).